plugin-list.xml

```diff
@@ -66,4 +66,15 @@
         <weblink>https://github.com/olav-st/screencloud-shellscript</weblink>
         <download>https://github.com/olav-st/screencloud-shellscript/releases/download/v1.5/v1.5.zip</download>
     </plugin>
+    <plugin>
+        <name>Lutim (framapic/...)</name>
+        <shortname>lutim</shortname>
+        <description>Upload to your Lutim instance</description>
+        <author>Thor77</author>
+        <version>2.1.0</version>
```

Comment:
Latest version is now `2.2.1´ (including the linked PR and a requests dependency update)

Reply:
Hello 

Thanks a lot, i've updated PR with this new version